modules/shortlinks.php

```diff
@@ -81,7 +81,7 @@ function wpme_get_shortlink( $id = 0, $context = 'post', $allow_slugs = true ) {
 	if ( empty( $type ) )
 		return '';
 
-	$url = 'http://wp.me/' . $type . wpme_dec2sixtwo( $blog_id ) . '-' . $id;
+	$url = 'https://wp.me/' . $type . wpme_dec2sixtwo( $blog_id ) . '-' . $id;
```

Comment:
We can't do this. wp.me will redirect based on the protocol and this makes the assumptions that all sites using it will be https.